### PR TITLE
fix for missing images when not using list view

### PR DESCRIPTION
**Pull request by leejordan**
**Summary**

The recently added list view caused a breaking change to non list view product tiles. This PR fixes that

**Notable Changes**

Addition of a distinct new class specifically for when using background images `.grid__panel-header-img-bg`

**QA**

Visit `/layouts/product-tiles` and ensure that when below 480px viewport size, the larger first product in the list has a visible product image

**Screenshots**

screen shot 2016-11-28 at 10 35 33

### CODE REVIEW

File: less/modules/panels.less [hunk 1]
```diff
@@ -157,23 +171,11 @@ a.grid__panel {
...
-        .background-cover();
-
-        &:after {
-            content: '';
-            display: block;
-            padding-top: 71%;
-        }
-    }
-
     @media (max-width: @screen-xs-min) {
```
Comment (notlee) [inline less/modules/panels.less]: Shouldn't this be `@media (max-width: @screen-xs-max) {` ?

File: site/docs/views/layouts_product-tiles.hbs [hunk 2]
```diff
@@ -82,11 +82,36 @@
                     <div class="row">
                         <ul class="grid">
 
+                            <li class="grid__col-xs-6 grid__col-md-4 grid__panel">
```
Comment (notlee) [inline site/docs/views/layouts_product-tiles.hbs]: "sour cream and onion cashews and garlic crostini" appears broken. Is it a mistake to leave it in when products are added dynamically below?

screen shot 2016-11-28 at 15 34 16

File: site/docs/views/patterns_panels.hbs [hunk 3]
```diff
@@ -115,14 +115,5 @@
             {{{ getFile 'site/docs/views/examples/panels/panel-grid.html' }}}
         {{/codeBlock}}
 
-        <h4 class="cd-text cd-title">panel grid with clickable panels</h4>
```
Comment (notlee) [inline site/docs/views/patterns_panels.hbs]: Are these removed just because we haven't used panels in this way?

**Comment by notlee** [comment]
Oh also do you consider this a breaking change to Pistachio? We didn't have a suggest/documented use of background images for panel headers right? In which case I guess not.

File: site/docs/views/layouts_product-tiles.hbs (hunk 2 above)
Comment (leejordan) [inline site/docs/views/layouts_product-tiles.hbs]: This is intentional - often the first image of a page will have this class omitted and therefore appear larger than the rest. This just demonstrates that.

File: less/modules/panels.less (hunk 1 above)
Comment (leejordan) [inline less/modules/panels.less]: `@media (max-width: @screen-xxs-max) {` actually 👍

**Pull request merged**